<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageList</class>
 <widget class="QWidget" name="PageList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Typ frontów:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditType">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="MyTable" name="tableWidgetFronts">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="columnCount">
        <number>4</number>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>150</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Wysokość</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Szerokość</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ilość</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Dodatkowe inf.</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="labelFrontsCount">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>Ilość frontów: 0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxCount">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Licz fronty</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxCountM2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Licz m2</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxUnderline">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Podkreślenie</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonAdd">
         <property name="text">
          <string>Dodaj</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonInsert">
         <property name="text">
          <string>Wstaw</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonDelete">
         <property name="text">
          <string>Usuń</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyTable</class>
   <extends>QTableWidget</extends>
   <header>mytable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
